bin/stats

```diff
@@ -161,6 +161,7 @@ if __name__ == "__main__":
     from trellodog import Trellodog
     from optparse import OptionParser
     parser = OptionParser(version=VERSION, usage=usage)
+    parser.add_option("-d", "--days", action="store", type="int", dest="days", help="number of days of activity to return")
```

Comment:
Right, but there's no way to expose that via CLI. Setting a default here, even if it's the same, means we can communicate it in help messages, etc.

For DRYness sake, you can make the default a class variable of the Trellodog class, and reference it both here, and in the method signature.